src/core-chrome-proxy.js

```diff
@@ -75,16 +75,14 @@ var chromeProxy = {
 	compileRules: function (proxyRulesList) {
 		if (!proxyRulesList || !proxyRulesList.length)
 			return [];
-		var result = [];
-
-		for (let i = 0; i < proxyRulesList.length; i++) {
-			let rule = proxyRulesList[i];
+		let result = [];
 
+		for (let rule of proxyRulesList) {
```

Comment:
Please do not use "for..of" in "core-chrome-proxy.js" and "core-firefox-proxy.js" other places is fine.
Proxy scripts run in the main thread and blocks the request. The script should be fast.
According to my tests "for..of" is 10% to 20% slower than simple "for" loops.
A benchmark for reference:
https://www.incredible-web.com/blog/performance-of-for-loops-with-javascript/